src/custom-types.json

```diff
@@ -813,6 +813,14 @@
       "https://en.wikipedia.org/wiki/Processing_(programming_language)"
     ]
   },
+  "text/x-rst": {
+    "compressible": true,
+    "extensions": ["rst"],
+    "sources": [
+      "https://docutils.sourceforge.io/rst.html",
+      "https://en.wikipedia.org/wiki/ReStructuredText"
```

Comment:
You can remove this, as links to Wikipedia re not accepted (see instructions at https://github.com/jshttp/mime-db/blob/master/README.md#contributing)